lib/rule-context.js

```diff
@@ -145,7 +157,9 @@ RuleContext.prototype = {
             location,
             message,
             opts,
-            this.meta
+            fix,
```

Comment:
`fix` is always `null` at this point. I had to add it so the following parameters are in the correct positions. I think it was just an oversight from when the fix option was added, but want to be sure.